tpl/tplimpl/embedded/templates/_default/rss.xml

```diff
@@ -32,7 +32,13 @@
       <pubDate>{{ .Date.Format "Mon, 02 Jan 2006 15:04:05 -0700" | safeHTML }}</pubDate>
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
-      <description>{{ .Summary | html }}</description>
+      <description>
+      {{- if .Site.Config.Services.RSS.FullContent -}}
+      {{ .Content | html }}
+      {{- else -}}
+      {{ .Summary | html }}
```

Comment:
`{{ .Content | transform.XMLEscape | safeHTML }}` & `{{ .Content | transform.XMLEscape | safeHTML }}`  is a more safe approach